### ISSUE TITLE
About the performance for Kitti testset.

### ISSUE THREAD

**Issue by tjulyz** [HKUST-Aerial-Robotics/Stereo-RCNN#57]
Hi,
I have carefully read the paper and the performance in your paper. From the leaderboard in Kitti, the performance is about 23% for hard subset.
![image](https://user-images.githubusercontent.com/18508335/68196203-30a89080-fff3-11e9-8c20-c3787dfe6df7.png)
But in your paper, the performance is about 28% for hard testset.
So could you please tell how to improve the test performance from 23% to 28%?
Have you used all the training data (i.e., training subset and validation subset) for training the final network when reporting the test performance?
Thanks a lot!

**Comment by PeiliangLi** [comment]
Hi, seems like the KITTI has changed the benchmark recently. You can check the details in the kitti official website, and previous results are in:http://www.cvlibs.net/datasets/kitti/backups/2019_10_08_10_46_41_object_3d.html

**Comment by tjulyz** [comment]
Thanks a lot!
So have you used all the training data (7000+) to get the final model for testset or just used 3000+ training data?

**Comment by tjulyz** [comment]
I have tried to submit the test results to the server by using the model trained on 3000+ training set. But only achieved 
![image](https://user-images.githubusercontent.com/18508335/68567028-f3308100-0492-11ea-84b4-21b499f8cf8b.png)
which has a large margin with the leadboard result as follows
![image](https://user-images.githubusercontent.com/18508335/68566979-d431ef00-0492-11ea-8b03-ac88a9697f06.png)
So could you please give me some advice for performance reproduction and fair comparison?

**Comment by PeiliangLi** [comment]
Hi, @tjulyz sorry for the late reply.
I use all the training data to get the kitti test results and the epoch should also be reduced (might be around 8 epochs). 
A simple Gaussian noise was also added to the input image. But I am not sure if this helps because we didn't do an ablation study due to the submission time limits.

**Comment by tjulyz** [comment]
Thanks a lot! Could you please tell how to train this final model? Have you change the learning rate or using finetuning?

**Comment by tjulyz** [comment]
Hi, I have tried the strategies you mentioned above (8 epochs and gaussian noise), but I still cannot reproduce the performance on test set (I can only achieve 18+% for hard set). Could you please provide more details? Thanks a lot!

**Comment by PeiliangLi** [comment]
Hi @tjulyz, I trace back to my previous experiment code and upload it to the exp branch. It should be the version for test server submission. Just run ./train.sh and test.sh to try it out.
I may not track the issue very frequently, feel free to drop me an email if you need immediate responses.
Thank you.

**Comment by tjulyz** [comment]
I will try it later. Thank you very much!
